From 211b393ab9b4b4644ef0dac82f5a1a4e417b3d49 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?No=C3=ABlie=20Ramuzat?= <noelie.ramuzat@laas.fr>
Date: Tue, 17 Aug 2021 13:30:04 +0200
Subject: [PATCH] [SotLoader] Remove useless boost::condition_variable cond

Remove useless condition variable which was never used (no call to wait()  function) and leads to errors when used with PAL packages.
---
 src/sot_loader.cpp | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/src/sot_loader.cpp b/src/sot_loader.cpp
index 2bf7ac4..80f4fd3 100644
--- src/sot_loader.cpp
+++ src/sot_loader.cpp
@@ -16,10 +16,6 @@
 // POSIX.1-2001
 #include <dlfcn.h>
 
-#include <boost/thread/condition.hpp>
-
-boost::condition_variable cond;
-
 using namespace std;
 using namespace dynamicgraph::sot;
 namespace po = boost::program_options;
@@ -98,7 +94,6 @@ void workThreadLoader(SotLoader *aSotLoader) {
     rate.sleep();
   }
   dataToLog.save("/tmp/geometric_simu");
-  cond.notify_all();
   ros::waitForShutdown();
 }
 
